modules/exploits/linux/http/trueonline_p660hn_v2_rce.rb

```diff
@@ -98,8 +98,8 @@ def exploit
     @cookie = rand_text_alpha_lower(7)
 
     res = send_request_cgi({
+      'method' => "GET",
```

Comment:
So is the code supposed to look something like this?? 

res = send_request_cgi({
        'uri'     => '/cgi-bin/index.asp?' + Rex::Text.encode_base64("#{datastore['USERNAME']}:#{datastore['PASSWORD']}"),
        'method'  => 'POST',
        'cookie'  => "SESSIONID=#{@cookie}",
        'vars_post' => {
          'Loginuser' => 'supervisor',
          'Prestige_Login' => 'Login'
        }
        'vars_get' => {
          #Some data => Some data
	  #Some data => Some data
        }
    })